app/src/pages/components/NavigationMap.jsx

```diff
@@ -0,0 +1,32 @@
+import { Header, MovieHeader } from './../header';
+import { Movies } from './../movies';
+import { Footer } from './../footer';
+import { Page404 } from './../404';

```

Comment:
BTW take a look at the React.lazy - and try to use

Reply:
It will be part of HW#11)